cli/cli/commands/enclave/inspect/user_services.go

```diff
@@ -29,16 +23,12 @@ const (
 	defaultEmptyIPAddrForAPIC     = ""
 )
 
-var (
-	colorizeRunning = color.New(color.FgGreen).SprintFunc()
-	colorizeStopped = color.New(color.FgYellow).SprintFunc()
-)
-
 func printUserServices(ctx context.Context, _ *kurtosis_context.KurtosisContext, enclaveInfo *kurtosis_engine_rpc_api_bindings.EnclaveInfo, showFullUuids bool, isAPIContainerRunning bool) error {
 	userServices := map[string]*kurtosis_core_rpc_api_bindings.ServiceInfo{}
 	if isAPIContainerRunning {
 		var err error
-		userServices, err = getUserServiceInfoMapFromAPIContainer(ctx, enclaveInfo)
+		allServicesMap := map[string]bool{}
+		userServices, err = user_services.GetUserServiceInfoMapFromAPIContainer(ctx, enclaveInfo, allServicesMap)
```

Comment:
The functions shared by the `enclave inspect` and `service inspect` have been moved to a CLI helper `user_services` module.